src/index.html

```diff
@@ -1,12 +1,316 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8"></meta>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"></meta>
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap" rel="stylesheet"></link>
+    <link rel="stylesheet" href="./styles/main.scss"></link>
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header_nav">
+      <a href="/" class="logo">
+        <img src="images/logo.png" alt="logo"></img>
+      </a>
+      <nav class="nav">
+        <a class="is-active nav_link" href="/">Apple</a>
```

Comment:
використовуй 2 нижніх підкреслення між назвою блока і назвою елемента
```suggestion
        <a class="is-active nav__link" href="/">Apple</a>
```